### ISSUE TITLE
Warning: findDOMNode is deprecated in StrictMode

### ISSUE THREAD

**Issue by zifik** [coreui/coreui-react#269]
I am getting this error with CModal v4.1.2. 

```
Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Transition which is inside StrictMode. Instead, add a ref directly to the element you want to reference.
```

```tsx
<CModal
  size="sm"
  alignment="center"
  backdrop="static"
  visible={loadingState.isBusy}
>
  <CModalBody className="text-center">
    <CSpinner component="span" />
    <h3>{loadingState.message}</h3>
  </CModalBody>
</CModal>
```

_Originally posted by @zifik in https://github.com/coreui/coreui-react/issues/127#issuecomment-1071663833_

**Comment by mrholek** [comment]
@zifik I will fix it next release.

**Comment by mrholek** [comment]
Fixed in v4.2.0
